cinema_service/urls.py

```diff
@@ -1,6 +1,8 @@
+from debug_toolbar.toolbar import debug_toolbar_urls
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-]
+    path("api/cinema/", include("cinema.urls", namespace="cinema")),
+] + debug_toolbar_urls()
```

Comment:
The `debug_toolbar_urls()` should only be included when `DEBUG` is `True`. Consider wrapping this line with a condition to check the `DEBUG` setting to prevent exposing the debug toolbar in a production environment.